hail/src/main/scala/is/hail/types/TypeWithRequiredness.scala

```diff
@@ -417,15 +417,15 @@ sealed abstract class RBaseStruct extends TypeWithRequiredness {
 }
 
 object RStruct {
-  def apply(fields: Seq[(String, TypeWithRequiredness)]): RStruct =
+  def apply(fields: IndexedSeq[(String, TypeWithRequiredness)])(implicit dummy: DummyImplicit): RStruct =
```

Comment:
prefer renaming when we get signature clashes like this. RStruct.fromNamesAndTypes?